api/client/samples/analogous_years/readme.md

```diff
@@ -0,0 +1,158 @@
+# Analogous Years
+Users can find similarity based ranks between a time period and a list of time periods 
+given by the time period ± 1 year, ± 2 years, ..., ± N years using `Analogous Years` application.
+For rank computation a user must provide:
```

Comment:
I think the intro should be a bit more accessible.  Perhaps something like:

"The `Analogous Years` application enables users to compare events from a set period of time, to those of the same date range in other years. The application will compute ranks of similarity between the specified period, and the same period from previous or future years. Multiple different inputs can be used in determining the ranks, but to compute these ranks, a user must provide:

Reply:
Copying your lines, thanks!